src/components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Footer, { footerText } from './Footer';
+import userEvent from '@testing-library/user-event';
+
+describe('Footer component', () => {
+  beforeEach(() => {
+    render(<Footer />);
+  });
+
+  it('should render main footer text', () => {
+    expect(screen.getByText(footerText)).toBeInTheDocument();
+  });
+
+  it('should open send email link', () => {
+    userEvent.click(screen.getByRole('link', { name: /Send us an email/i }));
+    expect(screen.getByText('Send us an email')).toBeInTheDocument();
+  });
+
+  it('should click GDPR link', () => {
+    userEvent.click(screen.getByRole('link', { name: /GDPR notice/i }));
+    expect(screen.getByText('GDPR notice')).toBeInTheDocument();
+  });
```

Comment:
I don't think these can be tested. You can click the links with userEvent.click but there is no way to check if an email window was opened or the GDPR pdf is downloaded.
You could instead use:
```
expect(screen.getAllByRole('link')[0].innerHTML).toBe('Send us an email');

...

expect(screen.getAllByRole('link')[1].innerHTML).toBe('GDPR notice');
```
which at least checks if they are links and the text is correct.

Reply:
Fixed